packages/payload/src/versions/saveVersion.ts

```diff
@@ -39,6 +39,8 @@ export const saveVersion = async ({
   let createNewVersion = true
   const now = new Date().toISOString()
   const versionData = deepCopyObjectSimple(doc)
+  versionData.updatedAt = now
```

Comment:
We need to adjust this based on `select`.
<img width="621" alt="image" src="https://github.com/user-attachments/assets/dd5e0869-52f3-4476-bafa-b6807db0a5f3">